### DISCUSSION TITLE
Polaris Figma UI Kit update?

### DISCUSSION THREAD

**Discussion by bochenn** [Shopify/polaris-react#9788] [General] [answered]
I see there are new ways in Figma to work (tokens, variants, specs) and streamline the handoff between designers and developers but the kit is old, and I guess is not reflecting the latest in the design system and react library.

Are you considering updating it?

Thanks!
L

**Comment by yurm04** [comment, marked as answer]
We recently updated the Figma UI Kit with the release of V12. [You can read more about it in this V12 guide](https://polaris.shopify.com/whats-new/version-12#figma-ui-kit).
